documentation/domains/Domain.md

```diff
@@ -89,6 +90,15 @@ The current status of the operation of the WebLogic domain. Updated automaticall
 | `serverStartPolicy` | string | The strategy for deciding whether to start a WebLogic Server instance. Legal values are NEVER, or IF_NEEDED. Defaults to IF_NEEDED. More info: https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle/startup/#starting-and-stopping-servers. |
 | `serverStartState` | string | The WebLogic runtime state in which the server is to be started. Use ADMIN if the server should start in the admin state. Defaults to RUNNING. |
 
+### Common Mount Volume
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `medium` | string | The emptyDir volume medium. This is an advanced setting that rarely needs to be configured. Defaults to unset, which means the volume's files are stored on the local node's file system for the life of the pod. |
+| `mountPath` | string | The common mount path. The files in the path are populated from the same named directory in the images supplied by each container in 'commonMount.containers'. Defaults to '/common'. |
+| `name` | string | The name of common mount volume. |
```

Comment:
Consider moving `name` first - it's the most important.  Also, consider making `mountPath` second.

Reply:
Add "Required."

Reply:
I have name and mountPath defined as first and second in CommonMountVolume.java but somehow generated `Domain.md ` is not adding it in that order (it seems to be sorted alphabetically). Any ideas on how to make name and mountPath appear first in the generated file?

Reply:
No. This is a minor issue - closing as resolved as we can ignore it.